dynawo/sources/Modeler/DataInterface/DYNDataInterfaceIIDM.cpp

```diff
@@ -386,8 +386,11 @@ DataInterfaceIIDM::importVoltageLevel(IIDM::VoltageLevel& voltageLevelIIDM) {
     components_[lcc->getID()] = lcc;
     lcc->setVoltageLevelInterface(voltageLevel);
   }
-
+#ifdef LANG_CXX11
```

Comment:
This is valid for any C++11 compilation, not only with clang. Moreover I am not sure we want to add macros like this that decreases substantially the readability of the code

Reply:
No this is not valid if every warning is considred as an error in clang:
error: prior to the resolution of a defect report against ISO C++11, local variable 'voltageLevel' would have been
      copied despite being returned by name, due to its not matching the function return type ('shared_ptr<DYN::VoltageLevelInterface>' vs 'shared_ptr<DYN::VoltageLevelInterfaceIIDM>')
      [-Werror,-Wreturn-std-move-in-c++11]

Reply:
No I mean that this piece of code is valid even for GCC C++ 11

Reply:
ok so I don't see any problem ?

Reply:
I see two action items here:
- do we want to add conditional macros about CXX 11 compilation? We need to discuss this with Adrien
- Please modifiy the conditonal macro so that it will be activated for any CXX 11 compilation (both GCC and CLANG)